pkg/oidc/verify_test.go

```diff
@@ -0,0 +1,202 @@
+package oidc
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/json"
+	"fmt"
+	"github.com/coreos/go-oidc"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"gopkg.in/square/go-jose.v2"
+)
+
+var _ = Describe("Verify", func() {
+	var (
+		ctx context.Context
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+	})
+
+	It("Succeeds with default aud behavior", func() {
+		result, err := verify(ctx, &IDTokenVerificationOptions{
+			AudienceClaim:  "aud",
+			ClientID:       "1226737",
+			ExtraAudiences: []string{},
+		}, payload{
+			Iss: "https://foo",
+			Aud: "1226737",
+		})
+
+		Expect(err).To(Succeed())
```

Comment:
This would be more idiomatic/readable if it were
```suggestion
		Expect(err).ToNot(HaveOccurred())
```